dev-qt/qtwebengine/qtwebengine-5.11.1.recipe

```diff
@@ -0,0 +1,207 @@
+SUMMARY="Qt WebEngine rendering library for dynamic web content"
+DESCRIPTION="Qt WebEngine provides functionality for rendering regions of dynamic web content. \
+The functionality in Qt WebEngine is divided into the following modules: \
+* QtWebEngineCore: Provides public API shared by both QtWebEngine and QtWebEngineWidgets \
+* QtWebEngine: Provides QML types for rendering web content within a QML application \
+* QtWebEngineWidgets: Provides a web browser engine as well as C++ classes to render \
+and interact with web content"
+
+HOMEPAGE="https://www.qt.io"
+COPYRIGHT="2015-2018 The Qt Company Ltd."
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://download.qt.io/official_releases/qt/5.11/5.11.1/submodules/qtwebengine-everywhere-src-5.11.1.tar.xz"
```

Comment:
```diff
-SOURCE_URI="https://download.qt.io/official_releases/qt/5.11/5.11.1/submodules/qtwebengine-everywhere-src-5.11.1.tar.xz"
+SOURCE_URI="https://download.qt.io/official_releases/qt/${portVersion%.*}/$portVersion/submodules/qtwebengine-everywhere-src-$portVersion.tar.xz"
```

This will allows the link to remain unchanged during bumps, minimizing the `diff` size